src/Illuminate/Support/Str.php

```diff
@@ -82,6 +82,25 @@ public static function camel($value)
     }
 
     /**
+     * It only takes first letter of the words in a sentence.
+     *
+     * @param  string  $value
+     * @param  bool $upperCase
+     * @return string
+     */
+    public static function acronym($value, $upperCase = false)
+    {
+        $letters = '';
+        $value = (string) preg_replace(['/[!@#$%^&*\/]/', '/\s+/'], ['', ' '], trim($value));
+
+        foreach (explode(' ', $value) as $word) {
+            $letters .= static::substr($word, 0, 1);
```

Comment:
To support multibyte encodings like UTF-8. With your method, you get the first byte, rather than the first character.
See [this stackoverflow question](https://stackoverflow.com/q/1972100/6386024)